.coderabbit.yaml

```
@@ -0,0 +1,20 @@
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
+language: "ko-KR"
+early_access: false
+reviews:
+  instructions: # 팀컨벤션 정의
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)